From: Sandro Tosi <morph@debian.org>
Date: Wed, 17 Jan 2018 20:20:58 -0500
Subject: disable intersphinx

---
 doc/conf.py | 4 ----
 1 file changed, 4 deletions(-)

--- a/doc/conf.py
+++ b/doc/conf.py
@@ -257,19 +257,6 @@
 
 
 intersphinx_mapping = {
-    'Pillow': ('https://pillow.readthedocs.io/en/stable/', None),
-    'cycler': ('https://matplotlib.org/cycler/', None),
-    'dateutil': ('https://dateutil.readthedocs.io/en/stable/', None),
-    'ipykernel': ('https://ipykernel.readthedocs.io/en/latest/', None),
-    'numpy': ('https://numpy.org/doc/stable/', None),
-    'pandas': ('https://pandas.pydata.org/pandas-docs/stable/', None),
-    'pytest': ('https://pytest.org/en/stable/', None),
-    'python': ('https://docs.python.org/3/', None),
-    'scipy': ('https://docs.scipy.org/doc/scipy/', None),
-    'tornado': ('https://www.tornadoweb.org/en/stable/', None),
-    'xarray': ('https://docs.xarray.dev/en/stable/', None),
-    'meson-python': ('https://mesonbuild.com/meson-python/', None),
-    'pip': ('https://pip.pypa.io/en/stable/', None),
 }
 
 
