From: Sandro Tosi <morph@debian.org>
Date: Wed, 17 Jan 2018 22:34:48 -0500
Subject: remove numpy scipy from reference_url

---
 doc/conf.py | 2 --
 1 file changed, 2 deletions(-)

--- a/doc/conf.py
+++ b/doc/conf.py
@@ -129,8 +129,6 @@ sphinx_gallery_conf = {
     'doc_module': ('matplotlib', 'mpl_toolkits'),
     'reference_url': {
         'matplotlib': None,
-        'numpy': 'https://docs.scipy.org/doc/numpy/',
-        'scipy': 'https://docs.scipy.org/doc/scipy/reference/',
     },
     'backreferences_dir': 'api/_as_gen',
     'subsection_order': gallery_order.sectionorder,
