--- a/doc/conf.py
+++ b/doc/conf.py
@@ -129,7 +129,6 @@
     'sphinxext.redirect_from',
     'sphinx_copybutton',
     'sphinx_design',
-    'sphinx_tags',
 ]
 
 exclude_patterns = [
@@ -196,11 +195,6 @@
         subsectionorder as gallery_order_subsectionorder)
     from sphinxext.util import clear_basic_units, matplotlib_reduced_latex_scraper
 
-if parse_version(sphinx_gallery.__version__) >= parse_version('0.17.0'):
-    sg_matplotlib_animations = (True, 'mp4')
-else:
-    sg_matplotlib_animations = True
-
 # The following import is only necessary to monkey patch the signature later on
 from sphinx_gallery import gen_rst
 
@@ -281,7 +275,6 @@
     'image_scrapers': (matplotlib_reduced_latex_scraper, ),
     'image_srcset': ["2x"],
     'junit': '../test-results/sphinx-gallery/junit.xml' if CIRCLECI else '',
-    'matplotlib_animations': sg_matplotlib_animations,
     'min_reported_time': 1,
     'plot_gallery': 'True',  # sphinx-gallery/913
     'reference_url': {'matplotlib': None, 'mpl_toolkits': None},
@@ -319,19 +312,6 @@
     logger = logging.getLogger('sphinx')
     logger.addFilter(gallery_image_warning_filter)
 
-# Sphinx tags configuration
-tags_create_tags = True
-tags_page_title = "All tags"
-tags_create_badges = True
-tags_badge_colors = {
-    "animation": "primary",
-    "component:*": "secondary",
-    "event-handling": "success",
-    "interactivity:*": "dark",
-    "plot-type:*": "danger",
-    "*": "light"  # default value
-}
-
 mathmpl_fontsize = 11.0
 mathmpl_srcset = ['2x']
 
